src/txn/txn_rollback_to_stable.c

```diff
@@ -749,7 +749,11 @@ __rollback_abort_newer_updates(
     if ((page = ref->page) == NULL || !__wt_page_is_modified(page)) {
         if (!__rollback_page_needs_abort(session, ref, rollback_timestamp)) {
             __wt_verbose(session, WT_VERB_RTS, "%p: page skipped", (void *)ref);
-#ifdef HAVE_DIAGNOSTIC
+#if 0
+            /*
+             * FIXME: Enable the clean page verification once the eviction is able to evict the
+             * newly loaded clean pages of this tree to avoid cache full.
+             */
```

Comment:
Could we instead replace the call to `__wt_page_release` with a call to `__wt_page_release_evict`? Is there a reason that immediate eviction would fail?

Reply:
There are other techniques as well. Possibly calling `__wt_page_evict_soon` prior to releasing the reference, or setting the `WT_SESSION_READ_WONT_NEED` flag for the entirety of rollback_to_stable, or passing in `WT_READ_WONT_NEED` to the page_in call.

Reply:
Thanks. `__wt_page_release_evict` should work. All other eviction logic doesn't work on that tree because the eviction is disabled for the tree until the rollback to stable is finished with it.